### ISSUE TITLE
Remove being able to set configuration

### ISSUE THREAD

**Issue by ureeves** [dusk-network/rusk-vm#373]
**Describe what you want implemented** 
Currently the user of the VM has the possibility to configure it upstream at runtime. This should be removed in favor of using only a default configuration set at build time.

**Describe "Why" this is needed**
We don't use the configuration upstream, and changing some options can lead to indeterminism, such as all the "gas per operation" options. 

**Describe alternatives you've considered**
It is possible to only allow a subset of configuration, however, since we don't use it upstream and this would lead to an increase in complexity it is the worse option.

**Additional context**
N/A

**Comment by ureeves** [comment]
Closing in favor of setting only once in #375
